test/presenters/document_collection_presenter_test.rb

```diff
@@ -103,6 +103,15 @@ class PresentedDocumentCollection < TestCase
 
       assert_nil grouped.first[:metadata][:public_updated_at]
     end
+
+    test "it returns nil if specfic document_type is in the disallowed_document_types list" do
```

Comment:
I think this test should check all the disallowed document types, not just simple_smart_answer